nestkernel/simulation_manager.cpp

```diff
@@ -706,6 +765,12 @@ nest::SimulationManager::update_connection_infrastructure( const thread tid )
     kernel().node_manager.set_have_nodes_changed( false );
   }
   kernel().connection_manager.unset_have_connections_changed( tid );
+
+#pragma omp barrier
```

Comment:
Probably only called once? So I guess there is not much overhead expected via this barrier.

Reply:
Yes, it's not frequently called. Keep this barrier.